config/initializers/omniauth.rb

```diff
@@ -8,7 +8,9 @@
 
   # CAS strategy
   if ENV['CAS_ENABLED'] == 'true'
-    cas_options = options
+    cas_options = {}
+    cas_options[:redirect_at_sign_in] = ENV['OAUTH_REDIRECT_AT_SIGN_IN'] == 'true'
```

Comment:
Drop that line, `ENV['OAUTH_REDIRECT_AT_SIGN_IN']` / `cas_options[:redirect_at_sign_in]` has been removed, in favor of `ENV['OMNIAUTH_ONLY']` which is handled independently